packages/react/src/index.ts

```diff
@@ -45,6 +45,10 @@ export {useRefObjectAsForwardedRef} from './hooks/useRefObjectAsForwardedRef'
 export {useResizeObserver} from './hooks/useResizeObserver'
 export {useResponsiveValue} from './hooks/useResponsiveValue'
 
+// Utils
+export {createComponent} from './utils/create-component'
```

Comment:
For stability, is this something that we feel like will be stable/unlikely to change without a breaking change or would it be helpful to have a "bake period" in drafts/experimental before promoting to stable?

Wasn't sure about the status of the helper and wanted to check-in 👍 Totally understand either way.

Reply:
It'll likely be removed when we upgrade to React 19. In lieu of that I think any changes can happen alongside major releases. I don't anticipate huge changes.